lib/generators/ruby_ui/install/templates/tailwind.config.js.erb

```diff
@@ -76,15 +67,8 @@ module.exports = {
         sm: "calc(var(--radius) - 4px)",
       },
       fontFamily: {
-        sans: ["var(--font-sans)", ...defaultTheme.fontFamily.sans],
+        sans: ["var(--font-sans)", 'ui-sans-serif', 'system-ui', 'sans-serif', '"Apple Color Emoji"', '"Segoe UI Emoji"', '"Segoe UI Symbol"', '"Noto Color Emoji"'],
       },
     },
   },
-  plugins: [
```

Comment:
I'll go into this in more detail when I look at javascript and component installers.